dokka-integration-tests/gradle/projects/it-sequential-tasks-execution-stress/build.gradle.kts

```diff
@@ -27,9 +27,9 @@ fun createTask(name: String) {
         dokkaSourceSets {
             moduleName.set("Some example")
             register("kotlin-stdlib-common") {
-                sourceRoots.from("src/main/java")
-                sourceRoots.from("src/main/kotlin")
-                samples.from("src/main/kotlin")
+                sourceRoots.from("src/common/java")
+                sourceRoots.from("src/common/kotlin")
+                samples.from("src/common/kotlin")
```

Comment:
Was it needed for something? It looks strange because this test applies `kotlin("jvm")`, so it's unclear why the source set should be named `common` (or is it not supposed to be the same as `commonMain`?)

Reply:
Oh, nevermind, the name of [the commit](https://github.com/Kotlin/dokka/pull/3390/commits/24b03240a80773ee3c24fb580f39814a57ecf425) that introduced it explains it

Adding a short comment here and in `it-basic-groovy` (in the similar place) might help a lot in the future, otherwise it's not very obvious that because the new source set is created, it cannot use the same source directory as the default source set, even though the default source set doesn't have any files in it